firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -1870,13 +1870,13 @@ extension String {
                 public static let ShowNonSponsoredSuggestionsTitle = MZLocalizedString(
                     key: "Settings.Search.Suggest.ShowNonSponsoredSuggestions.Title.v124",
                     tableName: "Settings",
-                    value: "Suggestions from %@",
-                    comment: "In the Search page of the Settings menu, the title for setting to enable Suggestions from Firefox. Placeholder is for the app name - Firefox.")
+                    value: "Suggestions from the Web",
```

Comment:
is "Web" supposed to be capitalized here?

Reply:
According to the design, yes.

![IMG_5808](https://github.com/mozilla-mobile/firefox-ios/assets/51127880/835801a1-bde2-4748-a5b3-9ad15b17e9a8)